integration_tests/web/test_admin_analytics.py

```diff
@@ -28,7 +28,7 @@ def tearDown(self):
     def test_sync(self):
         client = self.sync_client
 
-        response = client.admin_analytics_getFile(date="2020-10-20", type="member")
+        response = client.admin_analytics_getFile(date="2022-10-20", type="member")
```

Comment:
Since the old date no longer works for my sandbox org, let me change the date here